config/form_profile_mappings/22-1995.yml

```diff
@@ -1 +1,6 @@
-tours_of_duty: [military_information, tours_of_duty]
+veteranFullName: [identity_information, full_name]
+veteranSocialSecurityNumber: [identity_information, ssn]
+homePhone: [contact_information, us_phone]
+veteranAddress: [contact_information, address]
+email: [contact_information, email]
+toursOfDuty: [military_information, tours_of_duty]
```

Comment:
I'm not sure we should prefill this one on the 1995. We only ask them to fill in periods of service that are new since they applied for edu benefits, so this would be filling in a bunch of info that is probably not needed.

Reply:
Makes sense. I'll correct this.